src/components/table-of-contents-controller/table-of-contents-controller.tsx

```diff
@@ -16,36 +16,48 @@ export class TableOfContentsController {
   /**
    * Placeholder for search title filter
    * */
-  @Prop() titleFilterPlaceholder: string = "Search the table of contents";
+  @Prop() titleFilterPlaceholder: string = 'Search the table of contents';
+
+  /**
+   * If true, shows clear title filter button
+   */
+  @Prop() titleFilterClearable: boolean = false;
+
+  /**
+   * Additional classes added to title filter button
+   */
+  @Prop() titleFilterClearBtnClasses: string = '';
 
   /**
    * Additional classes added to Expand all button
    * */
-  @Prop() expandAllBtnClasses: string = "";
+  @Prop() expandAllBtnClasses: string = '';
 
   /**
    * Additional classes added to Collapse all button
    * */
-  @Prop() collapseAllBtnClasses: string = "";
+  @Prop() collapseAllBtnClasses: string = '';
 
   /**
    * Additional CSS classes added to the search filter input
    * */
-  @Prop() searchFilterInputClasses: string = "";
+  @Prop() searchFilterInputClasses: string = '';
```

Comment:
Maybe standardise to `titleFilterInputClasses` to match `titleFilter`?

Reply:
this is also considered to be a breaking change, projects currently using our package. Once we've updated la-web-components to a new version, if projects update la-web-components to that new version, they will need to change their attribute usage of `search-filter-input-classes` to `title-filter-input-classes`.
Open by laws, and africanlii currently use the toc-controller

Reply:
Yes, we'll have to update the places where we use this.